themes/codefor-theme/layouts/shortcodes/ziele/accordion-goals.html

```diff
@@ -7,7 +7,7 @@
 				<div class="row justify-content-center">
 					<div class="col-9 col-sm-5 col-xl-4 d-flex flex-column align-items-center">
 						<div class="px-md-4 w-100">
-							<img class="img-fluid p-2" src="/icons/{{ $element.icon }}">
+							<img class="img-fluid p-2" src="/icons/{{ $element.icon }}" alt="">
```

Comment:
I know, that the icons are picked, based on short codes. However, at least the name of the icon could give a hint if used as alt text.
```html
<img class="img-fluid p-2" src="/icons/{{ $element.icon }}" alt="{{ $element.icon }}">
```